pulley/fuzz/src/interp.rs

```diff
@@ -105,6 +105,15 @@ fn op_is_safe_for_fuzzing(op: &Op) -> bool {
         | Op::Xslteq32(Xslteq32 { operands, .. })
         | Op::Xult32(Xult32 { operands, .. })
         | Op::Xulteq32(Xulteq32 { operands, .. }) => !operands.dst.is_special(),
+        Op::PushFrame(_) | Op::PopFrame(_) => false,
+        Op::XPush32(XPush32 { src }) | Op::XPush64(XPush64 { src }) => !src.is_special(),
+        Op::XPop32(XPop32 { dst }) | Op::XPop64(XPop64 { dst }) => !dst.is_special(),
+        Op::XPush32Many(XPush32Many { srcs }) | Op::XPush64Many(XPush64Many { srcs }) => {
+            !srcs.into_iter().any(|src| src.is_special())
+        }
+        Op::XPop32Many(XPop32Many { dsts }) | Op::XPop64Many(XPop64Many { dsts }) => {
+            !dsts.into_iter().any(|dst| dst.is_special())
+        }
```

Comment:
These are only safe if we have enough stack space to push to or pop from, right?

So either we should make these not safe for fuzzing, or we should keep track of how much stack we've consumed and still have available at the point where this `op` is executed and gate `xpush`/`xpop`/`push_frame`/`pop_frame`'s fuzzing safety on whether there is enough stack to safely execute them.

Reply:
I'll return `false` for now, we can add a static analysis later